src/components/Aside/Aside.tsx

```diff
@@ -0,0 +1,28 @@
+import { ReactElement } from "react";
+import classnames from "classnames";
+
+import "./_aside.scss";
+
+type Props = {
+  children: ReactElement | ReactElement[];
```

Comment:
This should probably be `ReactNode` as it can be pretty much anything.

```suggestion
  children: ReactNode | ReactNode[];
```

Reply:
Ok